### PR TITLE
Added getter/setter info

### PR DESCRIPTION
**Pull request by felixsanz**
I tried to search for getter/setter in the readme but nothing was found. Added some info to make a clear statement about them

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2217,15 +2217,21 @@ Other Style Guides
 ## Accessors
 
   - [23.1](#23.1) <a name='23.1'></a> Accessor functions for properties are not required.
-  - [23.2](#23.2) <a name='23.2'></a> If you do make accessor functions use getVal() and setVal('hello').
+  - [23.2](#23.2) <a name='23.2'></a> Do not use JavaScript getters/setters as they cause unexpected side-effects and are harder to test, mantain and reason about. Instead, if you do make accessor functions, use getVal() and setVal('hello').
```
Comment (ljharb) [inline README.md]: `side effects` is two words, `s/mantain/maintain`, and please add the Oxford comma after "maintain".

File: README.md [hunk 2]
```diff
@@ -2217,15 +2217,21 @@ Other Style Guides
...
 
     ```javascript
-    // bad
+    // bad if using getter
```
Comment (ljharb) [inline README.md]: i'm not so sure these examples add anything tho

**Comment by felixsanz** [comment]
Sorry for the typos, fixed

**Comment by ljharb** [comment]
Looks great! Would you mind rebasing this down to a single commit?

**Comment by felixsanz** [comment]
@ljharb Git noob here. It is rebased correctly?

**Comment by ljharb** [comment]
@felixsanz almost! in the interim i added another commit to master :-) could you try once more?

**Comment by ljharb** [comment]
Duplicated by #757

**Pull request closed without merging**